bindings/python/fix_ssl.py

```diff
@@ -0,0 +1,26 @@
+import os
+import os.path
+import ssl
+import stat
+import subprocess
+import sys
+
+openssl_dir, openssl_cafile = os.path.split(
+    ssl.get_default_verify_paths().openssl_cafile)
+
+subprocess.check_call([sys.executable,
+        "-E", "-s", "-m", "pip", "install", "--upgrade", "certifi"])
+import certifi
+# change working directory to the default SSL directory
+os.chdir(openssl_dir)
+relpath_to_certifi_cafile = os.path.relpath(certifi.where())
+try:
+    os.remove(openssl_cafile)
```

Comment:
I don’t think it’s okay to replace the certificates store on evergreen hosts like this.

OpenSSL respects a `SSL_CERT_FILE` environment variable, that can be used to explicitly point to a certificate file – if that works, that seems like it could be a better approach?